Description: Check for existence of charsetp before using it.
Author: Jens Peter Secher <jps@debian.org>

Index: sml-mode/sml-mode.el
===================================================================
--- sml-mode.orig/sml-mode.el	2010-09-11 15:27:15.000000000 +0200
+++ sml-mode/sml-mode.el	2010-09-11 15:33:31.000000000 +0200
@@ -188,7 +188,7 @@
 (defconst sml-font-lock-symbols-alist
   (append
    ;; The symbols can come from a JIS0208 font.
-   (and (fboundp 'make-char) (charsetp 'japanese-jisx0208)
+   (and (fboundp 'make-char) (fboundp 'charsetp) (charsetp 'japanese-jisx0208)
 	(memq sml-font-lock-symbols '(t japanese-jisx0208))
 	(list (cons "fn" (make-char 'japanese-jisx0208 38 75))
 	      (cons "andalso" (make-char 'japanese-jisx0208 34 74))
