Description: Install into datadir
Author: Jens Peter Secher <jps@debian.org>

Index: sml-mode-5.0/Makefile
===================================================================
--- sml-mode-5.0.orig/Makefile	2013-07-17 19:05:25.278456373 -0400
+++ sml-mode-5.0/Makefile	2013-07-17 19:05:25.278456373 -0400
@@ -41,8 +41,8 @@
 elibdir = $(lispdir)/elib
 
 # the directory where you install the info doc
-infodir = $(prefix)/info
-docdir = $(prefix)/doc
+infodir = $(datadir)/info
+docdir = $(datadir)/doc
 
 EMACS	= emacs
 MAKEINFO= makeinfo
@@ -131,7 +131,7 @@
 	$(MKDIR) $(docdir)
 	$(CP) `find . -type f -name '*.dvi' -print` $(docdir)/
 
-install: install_elc install_info install_startup install_el
+install: install_elc install_info install_el
 
 clean:
 	$(RM) *~ core .\#* $(TEXEXTS)
